docs/safety.md

```diff
@@ -0,0 +1,113 @@
+
+The following items should be backed up regularly (e.g. hourly or even more
+frequently depending on the number of users/transactions of a system):
+
+* **If the default SQLite database is used:** Then all data is in the files
+  in the location `<tapddir>/data/<network>/tapd.db*` (usually `tapd.db`,
+  `tapd.db-wal` and `tapd.db-shm`), where `tapddir` is the following by default,
+  depending on your operating system:
+   * Linux/Unix: `~/.tapd`
+   * MacOS: `~/Library/Application Support/Tapd`
+   * Windows: `~/AppData/Roaming/Tapd`
+   * Umbrel: `${APP_DATA_DIR}/data/.tapd`
+   * Or, if either the `--tapddir` or `--datadir` flags or config options are
+   * set, then the file should be located there.
+* **If a Postgres database is used**: It is enough to create a backup of the
+  database configured as `--postgres.dbname` flag or config option.
+
+Optionally the copies of the proof files in `<tapddir>/data/<network>/proofs`
+can be backed up as well, but those are also all contained in the SQLite or
+Postgres database and are only on the filesystem for faster access.
+
+### Where are the private keys for assets stored?
+
+The `tapd` database does not store any private key material. It exclusively uses
+`lnd`'s wallet to derive keys for assets and their BTC anchoring transactions.
+The `tapd` database only stores the public key and derivation information in
+its database.
+
+The following cryptographic keys are derived from `lnd`'s wallet:
+* `internal_key`: The internal keys for BTC-level anchoring transaction outputs
+  that carry asset commitments.
+* `script_key`: The raw key for asset ownership keys, by default used as
+  BIP-0086 keys in the asset output.
+
+
+### Is it safe to restore from an outdated database backup?
+
+Yes. Since there is no penalty mechanism involved as in Lightning, there is no
+additional risk when restoring an outdated database backup.
+But of course, if the database backup is out of date, it might not contain the
+latest assets and access to those could still be lost.
+
+### Is it safe to open the `tapd` RPC port to the internet?
+
+There is normally no need to open the `tapd` RPC port (10029 by default) to the
+internet. Unless the intention is to run a public Universe server, then that is
+the port to expose. By default, all RPC methods (except for some non-sensitive
+Universe related calls) are protected by macaroon credentials.
+
+There are three flags/config options that should be evaluated though:
+* `--allow-public-uni-proof-courier`: If set, then access to the Universe based
+  proof courier methods is allowed without the requirement for a macaroon. That
+  means, any other nodes can use this `tapd` instance to transmit transfer
+  proofs from sender to receiver without needing any sort of permission
+  credential.
+* `--allow-public-stats`: If set, then access to Universe statistics RPC calls
+  are allowed without the requirement for a macaroon. This can be useful to
```

Comment:
```suggestion
* `--allow-public-stats`: If set, then access to Universe statistics RPC calls
  are allowed without the macaroon requirement. This can be useful to
```